src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.html

```diff
@@ -102,48 +119,48 @@ <h4 translate="KIBANA-ALERT_INDEX_PATTERN_DEPRECATED"></h4>
             </label>
             <select
               required
-              ng-options="opt.display for opt in index.nameIntervalOptions"
-              ng-model="index.nameInterval"
+              ng-options="opt.display for opt in ctrl.nameIntervalOptions"
```

Comment:
The point of the separation was to accomplish this note from the PR description:

> Separate user-specified variables for configuring the index pattern from controller variables, which just change the UI state.
>
> One of the issues I had with understanding this logic is that I really had a hard time figuring out which variables we're changing which define the index pattern we create. I tried to clarify this in the code.

But good point about the confusing nature of `ctrl`. I applied the "controllerAs" pattern to this to make it more idiomatic to Angular. What do you think?